package.json

```diff
@@ -15,7 +15,11 @@
       "name": "Themer",
       "description": "Themer theme"
     },
-    "asar": false,
+    "asar": true,
```

Comment:
Unfortunately `asar` needs to be `false` in order for `phantomjs` to work (a transitive dependency, used to render the wallpaper from SVG to PNG). Does the windows build fail with `"asar": false`?